restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -124,14 +125,14 @@ public Response putDoc(
   }
 
   @PUT
-  @Path("namespaces/{keyspace: [a-zA-Z_0-9]+}/collections/{collection}/{id}/{path: .*}")
+  @Path("namespaces/{namespace: [a-zA-Z_0-9]+}/collections/{collection}/{id}/{path: .*}")
```

Comment:
A bit confused. Shouldn't this be namespaces/{namespace-id}/collections/{collection-id}/{document-id}/{document-path}?

In fact, this seems to be missing a layer - the document bit.